db/schema.rb

```diff
@@ -85,11 +85,12 @@
     t.datetime "updated_at", null: false
     t.integer "row_order"
     t.bigint "project_id"
+    t.boolean "section", default: false
     t.boolean "complete", default: false
-    t.boolean "section"
-    t.bigint "assignee_id"
     t.datetime "deleted_at"
     t.datetime "due_date"
+    t.bigint "assignee_id"
```

Comment:
I see changes in schema but don't see any migrations

Reply:
it's fine. maybe settings of DB. same columns, just different order